app/models/administrator.rb

```diff
@@ -1,4 +1,12 @@
 class Administrator < ApplicationRecord
+  def self.authenticate(username, password)
```

Comment:
let's call this token, rather than password, as we're not encrypting/salting as a normal password would